lib/aruba/environment/windows_environment.rb

```diff
@@ -0,0 +1,11 @@
+require 'aruba/environment/basic_environment'
+
+module Aruba
+  class WindowsEnvironment < BasicEnvironment
+    def initialize
```

Comment:
What about something like this

```ruby
ENV.to_hash.inject({}) { |a, (k,v)| a[k.to_s.uppercase] = v }
```

If windows is case-insensitive for ENV, can we savely assume, that the code works for a program written in C, C# or whatever?